### ISSUE TITLE
Unable to scroll through message list.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#437]
```
Game version: 0.9.400


Operating System: Windows 7


Java runtime version: Java 7


Installed using the Launcher? Yes


Game language (en, hu): English


What steps will reproduce the problem?
1. Upon having more than 5 options on the 'SEND' list for messages
2.
3.


What is the expected output? What do you see instead?
When clicking on the down arrow (and when that works, up arrow), the list will 
scroll between the multiple options.  What instead occurs is that nothing 
happens.  There is no scroll or even acknowledgement of the button between 
presses

Please provide any additional information below.
I've noticed this occuring for some time, and it made an earlier build save 
game unfinishable because of hte inability to find the message to be sent to 
the Commander about the repeat infections and required quarantine operations.  
In latest builds, not nearly as much of an issue, since important messages seem 
to pop into the first 'page' of the list of options to send now, it might be a 
minor problem I'm itching about.
```

Original issue reported on code.google.com by `ganlon1...@hotmail.com` on 14 May 2012 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
Hi and thanks for reporting. 

Indeed, the buttons sometimes don't work, I'll fixit in the next release.

Until then, you can use the mouse wheel to scroll lists.
```

Original comment by `akarn...@gmail.com` on 14 May 2012 at 6:03
- Changed state: **Accepted**
- Added labels: **Priority-Low**, **Milestone-0.95**, **Component-UI**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.94.402
```

Original comment by `akarn...@gmail.com` on 17 May 2012 at 12:37
- Changed state: **SVN**

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 22 May 2012 at 2:34
- Changed state: **Fixed**
